### PR TITLE
feat: remove label import from csv

### PR DESCRIPTION
**Pull request by theodu**

### CODE REVIEW

File: docs/cli/index.md [hunk 1]
```diff
@@ -118,44 +118,37 @@ Ouput:
 
 ### Import labels to your project
 
-To import labels, provide a list of files or folders (you can also procide a csv file external_id and label's paths).
+To import labels, provide a list of files or folders.
 
 Label's files are json with the json_response to upload
```
Comment (PierreLeveau) [inline docs/cli/index.md:123]: may be a good time to fix this sentence:
Label files are JSON files containing labels in the Kili format: the value corresponding to the `jsonResponse` field of a label (see for example: https://docs.kili-technology.com/reference/json-classification-jobs).

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: docs/cli/index.md (hunk 1 above)
Comment (theodu) [reply docs/cli/index.md:123]: Agree!

File: docs/cli/index.md [hunk 2]
```diff
@@ -118,44 +118,41 @@ Ouput:
...
+jsonResponse field of a label
+(see [here](https://docs.kili-technology.com/reference/export-classification) for example). <br>
+File's name must be equal to asset's external_id.
 
 ```
 kili project label \
-    --project-id $project_id \
-    --from-csv labels.csv
+    labels \
+    --project-id $project_id
```
Comment (Jonas1312) [inline docs/cli/index.md:132]: I find this part quite confusing. `label` and `labels` are too similar. We can replace `labels` by some filenames directly

File: docs/cli/index.md [hunk 3]
```diff
@@ -118,44 +118,41 @@ Ouput:
...
+40 labels have been successfully imported
 ```
 
 If you have run a pre-annotation model, you can also import labels as predictions.
 These labels will be seen as pre-annotation in the labeling interface.
 
 ```
 kili project label \
+    labels \
     --project-id $project_id \
```
Comment (Jonas1312) [inline docs/cli/index.md:147]: same here

File: docs/cli/index.md (hunk 2 above)
Comment (theodu) [reply docs/cli/index.md:132]: it was to show that we can directly give a folder. in the demo folder, the folder name is labels. I will rename it in the

Comment (theodu) [reply docs/cli/index.md]: I renamed it ground_truths

Comment (theodu) [reply docs/cli/index.md]: folder with new name reuploaded to GCS

**Review by PierreLeveau — APPROVED** [review]

**Review by Jonas1312 — APPROVED** [review]

**Pull request merged**